powerlibs/django/restless/models.py

```diff
@@ -82,26 +82,24 @@ def serialize_model(obj, fields=None, include=None, exclude=None, fixup=None):
     def getvalueof(f):
         return getattr(obj, fieldmap.get(f, f))
 
-    if fields is None:
-        fields = list(fieldmap.keys())
-    else:
-        fields = list(fields)
+    fields = list(fields) if fields else list(fieldmap.keys())
 
-    if exclude is not None:
+    if exclude:
         fields = [f for f in fields if f not in exclude]
 
-    if include is not None:
-        for i in include:
-            if isinstance(i, tuple) or (isinstance(i, six.string_types)):
-                fields.append(i)
+    include = include or []
+    for i in include:
+        if isinstance(i, tuple) or (isinstance(i, six.string_types)):
```

Comment:
Pode ser 
isinstance(i, (tuple, six.string_types))

Reply:
Yeap. Sorry for that. Fixing.